tests/test_audio_amp_mute.py

```
@@ -0,0 +1,94 @@
+
+    def test_unmute_mute_noop_when_no_pinpong(self):
+        amp = board_mod.AudioAmpMute()
+        # 不应抛出异常
+        amp.unmute()
+        amp.mute()
+        amp.force_mute()
+        self.assertFalse(amp._available)
+
+
+class TestAudioAmpMuteWithFakePin(unittest.TestCase):
+    """模拟引脚，验证静音/解除静音电平与引用计数。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)